File: backend/CheckHash.java
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
public class CheckHash {
  public static void main(String[] args) {
    BCryptPasswordEncoder enc = new BCryptPasswordEncoder();
    String hash = "/A8pLDnm3HCBFhnzoQWlc/do5sVxzQa";
    for (String pw : new String[]{"password123", "password", "admin123", "Admin@123", "admin", "Admin@123!"}) {
      System.out.println(pw + " => " + enc.matches(pw, hash));
    }
  }
}

File: backend/check_hash.py
import bcrypt
h = b'$2a$10$j2FvC232mgVpfrdw1eTqeu/A8pLDnm3HCBFhnzoQWlc/do5sVxzQa'
for pw in [b'password123', b'password', b'admin123', b'Admin@123', b'admin', b'Admin@123!']:
    print(pw.decode(), bcrypt.checkpw(pw, h))
